NTupleMaker/bin/SynchNTupleProducer_2017.cpp

```diff
@@ -1335,25 +1319,6 @@ for (Long64_t iEntry=0; iEntry<numberOfEntries; iEntry++) {
 					  TMath::Sqrt( otree->met*TMath::Sin(otree->metphi)*otree->met*TMath::Sin(otree->metphi) +
 				          otree->met*TMath::Cos(otree->metphi)*otree->met*TMath::Cos(otree->metphi)));
 
-      if (ch == "mt" && checkDuplicateMuons && duplicatemuons.size()!=0 && isData){
-	// add to the MET LV the duplicate muons pt
-	TLorentzVector duplicateMuonsLV;
-	float duplicatemu_px = 0; 
-	float duplicatemu_py = 0; 
-	for (unsigned int i=0; i<duplicatemuons.size(); i++){
-	  duplicatemu_px += analysisTree.muon_px[duplicatemuons[i]];
-	  duplicatemu_py += analysisTree.muon_py[duplicatemuons[i]];
-	}
-	// add duplicate muons pT to MET
-	float tot_px = metLV.Px()+ duplicatemu_px;
-	float tot_py = metLV.Py() + duplicatemu_py;
-	metLV.SetPx(tot_px);
-	metLV.SetPy(tot_py);
-	// save to otree
-	otree->met= metLV.Pt();
-	otree->metphi = metLV.Phi();
-      }
-      counter[12]++;
 
```

Comment:
Here it seems you removed the possibility to check for duplicate muons, can you please explain why this is done?

Reply:
In the AN it is pointed that only for 2016 we need to check for duplicate muons.
I asked explicitly in a meeting if it is not overlooked. It is not, and if I remember correctly for the (ultra) legacy it is also not needed for 2016 anymore

Reply:
thus better remove the code then, since this is 2017 tupler. Important: we need to disucss in HiggsCP meeting what data we intend to use at end of the day, if legacy perhaps we can remove also for 2016.

Reply:
The filter can be removed since it is not needed for 2016 legacy nor for 2017 and also not for 2018.